../../../../CPP/7zip/Archive/Zip/ZipItem.cpp:429:7: error: 'bzero' was not declared in this scope

--- p7zip-17.05/CPP/7zip/Archive/Zip/ZipItem.cpp.old	2024-11-16 11:13:09.728242599 -0500
+++ p7zip-17.05/CPP/7zip/Archive/Zip/ZipItem.cpp	2024-11-16 11:14:00.478602980 -0500
@@ -5,6 +5,10 @@
 #include <locale.h>
 #endif
 
+#ifdef __sun__
+#include <strings.h>
+#endif
+
 #include "StdAfx.h"
 
 #include "../../../../C/CpuArch.h"
